api/src/main/resources/cb_review_api.yaml

```diff
@@ -241,6 +241,57 @@ paths:
           schema:
             $ref: "#/definitions/ParticipantDataListResponse"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/review/{cohortId}/{cdrVersionId}/participants/{participantId}/master:
+    parameters:
+      - $ref: '#/parameters/workspaceNamespace'
+      - $ref: '#/parameters/workspaceId'
+      - $ref: '#/parameters/cohortId'
+      - $ref: '#/parameters/cdrVersionId'
+      - $ref: '#/parameters/participantId'
+    post:
+      tags:
+        - cohortReview
+      description:  >
+            Returns a collection of all domain data for the specified params for a participant. This endpoint
+            does pagination based on page, limit, order and column.
+      operationId: "getMasterParticipantData"
+      parameters:
+        - in: body
+          name: request
+          required: true
+          description: request body for getting list of participant data.
+          schema:
+            $ref: "#/definitions/PageFilterRequest"
+      responses:
+        200:
+          description: A collection of ParticipantConditions.
+          schema:
+            $ref: "#/definitions/MasterParticipantDataListResponse"
+
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/review/{cohortId}/{cdrVersionId}/participants/{participantId}/detail/{domain}:
+    parameters:
+      - $ref: '#/parameters/workspaceNamespace'
+      - $ref: '#/parameters/workspaceId'
+      - $ref: '#/parameters/cohortId'
+      - $ref: '#/parameters/cdrVersionId'
+      - $ref: '#/parameters/participantId'
+    get:
+      tags:
+        - cohortReview
+      description: Returns detailed domain data for the specified params for a participant.
+      operationId: "getDetailParticipantData"
+      parameters:
+        - in: path
+          name: domain
```

Comment:
Doesn't this actually need to be an id (or equivalent) for a specific codes instance?

Reply:
Yeah.. i may just pass the id of the specific table instead of participant id.

Reply:
Just added dataId